docs/architecture/modern-web-apps-azure/common-web-application-architectures.md

```diff
@@ -93,8 +93,7 @@ The simplest approach to scaling a web application in Azure is to configure scal
 
 Applications that follow the Dependency Inversion Principle as well as the Domain-Driven Design (DDD) principles tend to arrive at a similar architecture. This architecture has gone by many names over the years. One of the first names was Hexagonal Architecture, followed by Ports-and-Adapters. More recently, it's been cited as the [Onion Architecture](https://jeffreypalermo.com/blog/the-onion-architecture-part-1/) or [Clean Architecture](https://8thlight.com/blog/uncle-bob/2012/08/13/the-clean-architecture.html). The latter name, Clean Architecture, is used as the name for this architecture in this e-book.
 
-> [!NOTE]
-> The term Clean Architecture can be applied to applications that are built using DDD Principles as well as to those that are not built using DDD. In the case of the former, this combination may be referred to as "Clean DDD Architecture".
+The eShopOnWeb reference application uses the Clean Architecture approach in organizing its code into projects. You can find a solution template you can use as a staring point for your own ASP.NET Core on GitHub at [github.com/ardalis/cleanarchitecture](https://github.com/ardalis/cleanarchitecture).
```

Comment:
Shouldn't this be on an official dotnet repo?

```suggestion
The eShopOnWeb reference application uses the Clean Architecture approach in organizing its code into projects. You can find a solution template you can use as a starting point for your own ASP.NET Core on the [ardalis/cleanarchitecture](https://github.com/ardalis/cleanarchitecture) GitHub repository.
```

Reply:
MS doesn't have one - this one is mine and I'm referring folks to it as the author of the ebook. Hopefully that's still OK.

Reply:
yep, it's fine. It's just that I've seen in the past work that was done as part of the e-book project pointing to someone's repo. Still not a problem, but those should be maintained by Microsoft.